packages/integration/lib/src/widgets/lists/feed/feed_list.dart

```diff
@@ -306,6 +306,7 @@ class _LMFeedListState extends State<LMFeedList> {
                     LMFeedFireAnalyticsEvent(
                       eventName: LMFeedAnalyticsKeys.postDeleted,
                       deprecatedEventName: LMFeedAnalyticsKeysDep.postDeleted,
+                      widgetSource: LMFeedWidgetSource.universalFeed,
```

Comment:
if feed list will be used for other feeds i.e: user_feed or user_created_comment_feed, then is it right to assign widgetSurce: LMFeedWidgetSource.universalFeed ?

Reply:
That should be customisable too, will change it

Reply:
I looked into the widget and it is for fetching universal feed list, just like you created one for `UserSavedPostList`.